crash-handler-process/platforms/util-win.cpp

```diff
@@ -302,9 +302,10 @@ void Util::setCachePath(std::wstring path)
 	appCachePath = path;
 }
 
-void Util::updateAppState(bool unresponsive_detected)
+void Util::updateAppState(Util::AppState detected)
```

Comment:
I would suggest "state" for the variable name, I think it explains better what it does.